test/testMultihypoFMD.jl

```diff
@@ -16,14 +16,14 @@ end
 
 function getSample( cf::CalcFactor{<:MyFactor})
   #
-  @warn "getSample(cf::CalcFactor{<:MyFactor},::Int) does not get hypo sub-selected FMD data"
-  @show DFG.getLabel.(cf.metadata.fullvariables)
+  @warn "getSample(cf::CalcFactor{<:MyFactor},::Int) does not get hypo sub-selected FMD data: $(DFG.getLabel.(cf.fullvariables))" cf.solvefor maxlog=1
   # @assert DFG.getLabel.(fmd_[1].fullvariables) |> length < 3 "this factor is only between two variables"
   return rand(cf.factor.Z, 1)
 end
 
 function (cf::CalcFactor{<:MyFactor})(z, X1, X2)
-  @assert DFG.getLabel.(cf.metadata.fullvariables) |> length < 3 "this factor is only between two variables"
+  length(cf.fullvariables) < 3 && @warn("this factor is only between two variables. solvefor=$(cf.solvefor)", maxlog=1)
```

Comment:
```suggestion
  @assert length(cf.fullvariables) < 3 "this factor is only between two variables. solvefor=$(cf.solvefor)"
```

This error is important in testing

Reply:
Jip, I fixed it, was just to test with and help me understand.